test/non-interference/http-e2e.js

```diff
@@ -49,7 +49,16 @@ var test_glob = semver.satisfies(process.version, '0.12.x') ?
 
 // Run tests
 console.log('Running tests');
-var gcloud_require = 'require(\'' + path.join(__dirname, '..', '..') +
+var gcloud_require =
+    'var proxyquire = require(\'' +
+    path.join(__dirname, '..', '..', 'node_modules', 'proxyquire') +
```

Comment:
Is that because we don't run this test on windows? I would guess that `require` does not handle mixed delimiter path names.